code/modules/antagonists/slaughter/imp_antag.dm

```diff
@@ -18,3 +19,10 @@
 	newobjective.explanation_text = "Try to get a promotion to a higher devilish rank."
 	newobjective.owner = owner
 	objectives += newobjective
+
+/datum/antagonist/imp/ui_static_data(mob/user)
+	. = ..()
+	var/list/data = list()
```

Comment:
setting the return value here only to explicitly return at the end of the statement is a bit redundant. Could you change that?